persistent-postgresql/Database/Persist/Postgresql/Util.hs

```diff
@@ -0,0 +1,30 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+module Database.Persist.Postgresql.Util (
+  mkPostgresUpdateText
```

Comment:
Since this module only has one function, let's just put it directly in `Database.Persist.Postgresql`